.circleci/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.11.2
+    steps:
+      - checkout
+      - run: npm install
+      - run: npm test
+      - run: npm run coveralls
```

Comment:
Please, remove this command. Travis will do the job.

Reply:
Following the CircleCI documentation version 2.0, this codes is necessary to make the builds. If you remove all this commands, CircleCI return a following error: ```jobs: build: required key [steps] not found```, so, on this part of the configuration, is needed at least one command.

So, if you go replace Travis to CircleCI, this commands ends up being necessary to lint, test and send the coveralls test results to Coveralls.io

Reply:
You're right, but I asked you to remove only this command: `- run: npm run coveralls`. Travis already sends the results to Coveralls and it doesn't need the token. :wink:

Travis configuration file:

```yaml
after_script:
  - npm run coveralls
```

You got it?